common/select.h

```diff
@@ -33,7 +32,7 @@ class Select
         TIMEOUT = 2,
```

Comment:
updating this enum with something like
`NO_TIMEOUT = -1`
will allow you to use 
`swsscommon.Select.NO_TIMEOUT`
in plugin

Reply:
Just aware the default value of timeout is -1, so we don't need this NO_TIMEOUT anymore.

Reply:
See my updates for another patch..

Reply:
https://github.com/kevinwangsk/sonic-buildimage/pull/1/files